android/src/main/java/com/appcuesreactnative/AppcuesReactNativeModule.kt

```diff
@@ -14,7 +14,7 @@ import kotlinx.coroutines.launch
 
 class AppcuesReactNativeModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {
 
-    private lateinit var implementation: Appcues
+    private var implementation: Appcues? = null
```

Comment:
as an Idea here we could change the "get() = {}" here to check if its null, if it is we print a terminal warning explaining that a call to the native API is being ignored because the instance is not ready. and link to the docs where they can setup the sync/await strategy.